ReactiveObjCBridge/ObjectiveCBridging.swift

```diff
@@ -195,6 +224,34 @@ extension SignalProtocol {
 	}
 }
 
+extension SignalProtocol where Self.Value: OptionalProtocol {
+	/// Create a `RACSignal` that will observe the given signal.
+	///
+	/// - note: Any `interrupted` events will be silently discarded.
+	///
+	/// - returns: `RACSignal` instantiated from `self`.
+	public func toRACSignal() -> RACSignal {
```

Comment:
I'm afraid not (at least without creating a third private function that both the original implementations and overloads use) because of the line `subscriber.sendNext(value.optional)`, which differs from the original implementation's `subscriber.sendNext(value)`.

Side note: this has me wishing that there were a way to specify in the unit tests exactly which overload is being used (rather than leaving it up to  the compiler to choose). It would at least allow us to be more expressive in our tests, and guarantee that we're testing the correct method.